_posts/2017-11-30-totw-130.md

```diff
@@ -0,0 +1,185 @@
+---
+title: "Tip of the Week #130: Namespace Naming"
+layout: tips
+sidenav: side-nav-tips.html
+published: false
+permalink: tips/130
+type: markdown
+order: "130"
+---
+
+Originally posted as totw/130 on 2017-02-17
+
+By Titus Winters (titus@google.com)
+
+*The precision of naming takes away from the uniqueness of seeing* — Pierre
+Bonnard
```

Comment:
Include author in italics quote

Reply:
I don't think we've been doing that. You sure?